sushy_oem_idrac/resources/manager/manager.py

```diff
@@ -1,3 +1,5 @@
+# Copyright (c) 2019 Dell Inc. or its subsidiaries.
```

Comment:
There might be other files carrying Apache license note. If you want to add Dell copyright, may be we should do that for other files as well. May be all at once?